compiler/rustc_codegen_ssa/src/traits/backend.rs

```diff
@@ -57,6 +57,9 @@ impl<'tcx, T> Backend<'tcx> for T where
 }
 
 pub trait CodegenBackend {
+    /// Locale resources for diagnostic messages - a string the content of the Fluent resource.
+    fn locale_resource(&self) -> &'static str;
```

Comment:
Is this called before or after init? If called before can you add a comment to this effect? Maybe move it before such that init can emit translatable diagnostics.

Reply:
Fixed, it's called before `init`.